_includes/manuals/2.2/1.2_release_notes.md

```diff
@@ -16,14 +12,252 @@ aren't familiar with all the methods Foreman provides. Documentation is availabl
+* Move user-dropdown to PF4 ([#30232](https://projects.theforeman.org/issues/30232))
+* User role with long name without spaces overlap to another column ([#30178](https://projects.theforeman.org/issues/30178))
+* UserDropdowns.scss is not used and is a duplicate ([#30139](https://projects.theforeman.org/issues/30139))
+* Host Detail Page Redesign  ([#30044](https://projects.theforeman.org/issues/30044))
+* follow naming convention for the auth sources tab ([#29958](https://projects.theforeman.org/issues/29958))
+* Bad name for numericInput component ([#29902](https://projects.theforeman.org/issues/29902))
+* Add cpu/cores/socket react cmponent ([#29900](https://projects.theforeman.org/issues/29900))
+* Update breadcrumbs to patternfly v4 ([#29332](https://projects.theforeman.org/issues/29332))
+* Update foreman fonts to patternfly v4 ([#29331](https://projects.theforeman.org/issues/29331))
+* ERF42-4995 - Invalid authenticity token (Session timed out at login screen) ([#10577](https://projects.theforeman.org/issues/10577))
+
+#### foreman-installer script
+* make it possible to brand katello-certs-check ([#30251](https://projects.theforeman.org/issues/30251))
+* Remove MongoDB Storage Engine Migration Hook ([#30006](https://projects.theforeman.org/issues/30006))
+* Remove the dependency on foreman-maintain ([#29963](https://projects.theforeman.org/issues/29963))
+* Drop --upgrade flag from katello hooks ([#29296](https://projects.theforeman.org/issues/29296))
+
+#### foreman-tasks
+* During smart proxy sync display the filter is set to "resource_id = 3 AND resource_type = SmartProxy", the tasks are displayed twice and Foreman Task id is set to "zero" ([#30376](https://projects.theforeman.org/issues/30376))
+
+*A full list of changes in 2.2.0 is available via [Redmine](https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f[]=cf_12&op[cf_12]=%3D&v[cf_12][]=1234)*
 
 ### Deprecations
 
 ### Upgrade warnings
 
-* Trends and statistics are now moved to separate plugin. See [manual](/plugins/foreman_statistics) of the new Statistics plugin if you wish continue using it.
+* The `--upgrade` installer flag has been removed. When running the installer as part of the upgrade procedure it is not required.
+* Trends and statistics are now moved to a separate plugin. See the [manual](/plugins/foreman_statistics) of the new Statistics plugin if you wish continue using them.
 * TFTP integration is now [disabled by default](https://community.theforeman.org/t/disable-foreman-proxy-tftp-support-by-default/18892). Upgrades are unaffected. To enable TFTP integration, `--foreman-proxy-tftp true` must be passed to the installer command.
+* Puppet run button has been removed from the host detail page. To trigger one-time puppet agent runs, please use the Remote Execution plugin instead. ([More info](https://community.theforeman.org/t/dropping-run-puppet-button-with-a-replacement-using-remote-execution-plugin/12781/25))
```

Comment:
Isn't it `Run Puppet`? It also used to be on the host list in the bulk actions, not just on the detail page.
```suggestion
* The Run Puppet functionality based on puppetrun has been removed. To trigger one-time puppet agent runs, please use the Remote Execution plugin instead. ([More info](https://community.theforeman.org/t/dropping-run-puppet-button-with-a-replacement-using-remote-execution-plugin/12781/25))
```